### ISSUE TITLE
Organize documentation about Application and WindowSpecification objects better

### ISSUE THREAD

**Issue by vasily-v-ryabov** [pywinauto/pywinauto#873]
at the root folder and in the OS specific implementations.

**Comment by airelil** [comment]
As a first step, I can fix the documentation generation in atspi branch

**Comment by vasily-v-ryabov** [comment]
I've merged the first PR. What I could find is that method `start` contains just list of params, but there is no doc string (at the end of page): https://airelil-pywinauto.readthedocs.io/en/atspi/code/pywinauto.application.html

**Comment by airelil** [comment]
Yeah... That's because now we parse the Linux implementation of Application. I'll update the doc strings in `python.linux.applicaiont.Application`, but now wee need to keep it in mind that Linux version of any class will be the main reference in the doc.
